plugins/osdn/node.go

```diff
@@ -139,6 +148,46 @@ func (node *OsdnNode) Start() error {
 	return nil
 }
 
+const cniConfigPath = "/etc/cni/net.d/80-openshift-sdn.conf"
+
+func writeCniConfig() error {
+	const config string =`{
+  "cniVersion": "0.1.0",
+  "name": "openshift-sdn",
+  "type": "openshift-sdn"
+}`
+
+	return ioutil.WriteFile(cniConfigPath, []byte(config), 0600)
```

Comment:
As commented above, originally I was going to write the cluster info to this file as well, but kubelet's CNI plugin needs this file immediately, and we don't have cluster info til later.  So you're right, this can now be dropped statically on the filesystem.